tests/dialects/test_riscv.py

```diff
@@ -77,3 +78,23 @@ def test_csr_op():
     riscv.CsrrsiOp(
         csr=csr, immediate=IntegerAttr(1, i32), rd=riscv.Registers.A2
     ).verify()
+
+
+def test_comment_op():
+    comment_op = riscv.CommentOp("my comment")
+
+    assert comment_op.comment.data == "my comment"
+
+    code = riscv_code(ModuleOp([comment_op]))
+    assert code == "    # my comment\n"
+
+
+def test_return_op():
+    return_op = riscv.EbreakOp(comment="my comment")
+
+    assert return_op.comment is not None
+
+    assert return_op.comment.data == "my comment"
+
+    code = riscv_code(ModuleOp([return_op]))
```

Comment:
I would add a test when the comment gets too big ;)

Reply:
You mean when the line before the comment gets too big?

Reply:
The comment can be as long as you want